src/rest/UserProfilingEndpoint.ts

```diff
@@ -109,20 +104,8 @@ export class UserProfilingEndpoint {
      *
      * @param userId Id from which action history will be retrieve. (either visitId or user email).
      */
-    public getActions(userId: string): Promise<IActionHistory[]> {
+    public async getActions(userId: string): Promise<IActionHistory[]> {
         Assert.exists(userId);
-
-        let request = this.pendingGetActions[userId];
-
-        if (!request) {
-            request = this.buildGetActionRequest(userId);
-            this.pendingGetActions[userId] = request;
-        }
-
-        return request;
-    }
-
-    private async buildGetActionRequest(userId: string) {
```

Comment:
What's the reason for removing this functions functionality, and letting it skip straight to the `buildGetActionRequest` functionality?

Reply:
It's one of the caching functionnality that I removed.